src/pq_flash_index.cpp

```diff
@@ -1536,6 +1536,85 @@ template <typename T, typename LabelT> diskann::Metric PQFlashIndex<T, LabelT>::
     return this->metric;
 }
 
+template <typename T, typename LabelT> std::vector<T> PQFlashIndex<T, LabelT>::get_vector(std::uint64_t VID)
```

Comment:
VID should be in small letters?

Reply:
@philipBAdams could you please make the change Gopal requests

Reply:
This method no longer exists since I use your `read_nodes` method to get the vector coordinates instead